examples/benchmarks/pytorch_bert_large.py

```diff
@@ -3,25 +3,21 @@
 
 """Model benchmark example for bert-large."""
 
-from superbench.benchmarks import Platform, Framework, BenchmarkRegistry, BenchmarkContext
+from superbench.benchmarks import Framework, BenchmarkRegistry
 from superbench.common.utils import logger
 
 if __name__ == '__main__':
     # Create context for bert-large benchmark and run it for 120 * 2 seconds.
-    context = BenchmarkContext(
+    context = BenchmarkRegistry.create_benchmark_context(
         'bert-large',
-        Platform.CUDA,
```

Comment:
Do we remove it?

Reply:
No need to specify platform here. Since bert-large is registered for all platforms.

Reply:
`platform = Platform.CPU` would be clear as the example.

Reply:
Do you mean we will get all results when we call BenchmarkRegistry.launch_benchmark(context)?